src/app/common/default.controller.js

```diff
@@ -48,6 +50,37 @@
         userService.setStoredUser(self.user);
       }
 
+      function getBMI(event) {
+        calculate();
+        if (self.user.BMI<19) {
+          self.message = 'You body mass index is ' + self.user.BMI + '. You are slim.';
+        } else if (self.user.BMI >= 19 && self.user.BMI <= 25){
+          self.message = 'You body mass index is ' + self.user.BMI + '. You are fit.';
+        } else if (self.user.BMI<19){
+          self.message = 'You body mass index is ' + self.user.BMI + '. You are overweight.';
+        }
+          self.message += ' Also consider that your Basal metabolic rate is ';
+          self.message += self.user.BMR;
+          self.message += '. That means you need ';
+          self.message += self.user.BMR;
+          self.message += ' calories per day just to maintain basic body functions.';
```

Comment:
JSlinter told me it's too long.
-[x] refactored
https://github.com/VasylPosmit/health/pull/33